{"uuid":"8bc81baf-46f9-40ff-afa6-dc2171433671","historyId":"29a2698266a8af542e3bc65130ebbf10","fullName":"com.qa.tests.CompaniesPageTest.companyPageTest","labels":[{"name":"package","value":"com.qa.tests.CompaniesPageTest"},{"name":"testClass","value":"com.qa.tests.CompaniesPageTest"},{"name":"testMethod","value":"companyPageTest"},{"name":"parentSuite","value":"CogmentoSuite"},{"name":"suite","value":"CogmentoCompaniesPage"},{"name":"subSuite","value":"com.qa.tests.CompaniesPageTest"},{"name":"host","value":"DeeshaST123"},{"name":"thread","value":"26164@DeeshaST123.TestNG-tests-4(18)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"companyPageTest","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"org.apache.poi.openxml4j.exceptions.OpenXML4JRuntimeException: Fail to save: an error occurs while saving the package : The part /xl/worksheets/sheet5.xml failed to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@5e88be9b. Enable logging via Log4j 2 for more details.","trace":"java.lang.RuntimeException: org.apache.poi.openxml4j.exceptions.OpenXML4JRuntimeException: Fail to save: an error occurs while saving the package : The part /xl/worksheets/sheet5.xml failed to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@5e88be9b. Enable logging via Log4j 2 for more details.\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:50)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:146)\r\n\tat org.testng.internal.Parameters.handleParameters(Parameters.java:798)\r\n\tat org.testng.internal.Parameters.handleParameters(Parameters.java:740)\r\n\tat org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)\r\n\tat org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)\r\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:791)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1541)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:794)\r\n\tat org.testng.TestRunner.run(TestRunner.java:596)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:377)\r\n\tat org.testng.SuiteRunner.access$000(SuiteRunner.java:28)\r\n\tat org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)\r\n\tat org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)\r\n\tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)\r\n\tat java.base/java.lang.Thread.run(Thread.java:834)\r\nCaused by: org.apache.poi.openxml4j.exceptions.OpenXML4JRuntimeException: Fail to save: an error occurs while saving the package : The part /xl/worksheets/sheet5.xml failed to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@5e88be9b. Enable logging via Log4j 2 for more details.\r\n\tat org.apache.poi.openxml4j.opc.ZipPackage.saveImpl(ZipPackage.java:598)\r\n\tat org.apache.poi.openxml4j.opc.OPCPackage.save(OPCPackage.java:1496)\r\n\tat org.apache.poi.openxml4j.opc.OPCPackage.save(OPCPackage.java:1483)\r\n\tat org.apache.poi.openxml4j.opc.ZipPackage.closeImpl(ZipPackage.java:472)\r\n\tat org.apache.poi.openxml4j.opc.OPCPackage.close(OPCPackage.java:470)\r\n\tat org.apache.poi.ooxml.POIXMLDocument.close(POIXMLDocument.java:189)\r\n\tat org.apache.poi.xssf.usermodel.XSSFWorkbook.close(XSSFWorkbook.java:610)\r\n\tat com.qa.util.CommonFunctions.testData(CommonFunctions.java:154)\r\n\tat com.qa.tests.CompaniesPageTest.getData1(CompaniesPageTest.java:54)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:77)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:46)\r\n\t... 20 more\r\nCaused by: org.apache.poi.openxml4j.exceptions.OpenXML4JException: The part /xl/worksheets/sheet5.xml failed to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@5e88be9b. Enable logging via Log4j 2 for more details.\r\n\tat org.apache.poi.openxml4j.opc.ZipPackage.saveImpl(ZipPackage.java:587)\r\n\t... 35 more\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1676458047783,"stop":1676458047791}